web-client/src/presenter/actions/CaseDeadline/createCaseDeadlineAction.ts

```diff
@@ -25,16 +27,17 @@ export const createCaseDeadlineAction = async ({
     leadDocketNumber,
   };
 
-  const createCaseDeadlineResult = await applicationContext
+  const updatedCase = await applicationContext
     .getUseCases()
     .createCaseDeadlineInteractor(applicationContext, {
       caseDeadline,
     });
 
+  setAutomaticBlockedFieldsOnState(updatedCase, store);
```

Comment:
Just mimicking what we do with deleteCaseDeadlineAction to update the Blocked indicator without needing a refresh.